src/pages/User/TasksSession.js

```diff
@@ -0,0 +1,71 @@
+import React, {useEffect, useContext, useState} from 'react';
+import Paper from '@material-ui/core/Paper';
+import TasksTable from '../../components/TasksTable';
+import GetTasks from '../../utils/GetTasks.js';
+import { KPIStoreContext } from '../../contexts/KPIStore.js';
+import PropTypes from 'prop-types';
+
+export default function TasksSession({...props}) {
+  const {userid, sprintID, reload} = props;
+  const [loading, setLoading] = useState(true);
+  const { tickets: [tickets, setTickets] } = useContext(KPIStoreContext);
+  const assignTickets = tickets.filter((item) => item.user_id === userid && item.sprint_id === sprintID);
+
+  const fetchTasks = () => {
+    GetTasks(userid, sprintID).then((results) => {
+      const keyList = results.data.sprint_tickets.map((item) => item.key);
+      const ticketsMap = results.data.sprint_tickets.map((item) => {
+        // TODO BE return sprint_id. since 1 ticket might have more than 1 sprint_id
+        // item.sprint_id = '1,2,3'
+        item.sprint_id = sprintID;
```

Comment:
@TShida951 we need BE return sprint_id here,
Cause 1 ticket might belong to multiple Sprint

Reply:
![Screen Shot 2019-12-02 at 17 07 17](https://user-images.githubusercontent.com/7850988/69950804-9ae71f00-1526-11ea-8f8f-f0c9b0aa1b9b.png)